ros/src/actuation/vehicles/packages/ymc/node/g30esli_interface/g30esli_ros.cpp

```diff
@@ -16,7 +16,8 @@
 
 #include "g30esli_ros.h"
 
-G30esliROS::G30esliROS() : alive_(0)
+G30esliROS::G30esliROS()
+  : alive_(0), reset_command_(false), stop_keep_secs_(1.0), reset_keep_secs_(0.6), disable_reset_secs_(3.0)
```

Comment:
Do you need to initialize commands_ and status_?

Reply:
g30esli command and status are initialized in `g30esli`.